### ISSUE TITLE
Inference code keeps generating the same image

### ISSUE THREAD

**Issue by moeinheidari7829** [FoundationVision/VAR#61]
Dear everyone,

Has anyone had the experience of generating images using   "recon_B3HW = var.autoregressive_infer_cfg(B=1, label_B=torch.tensor([label], device=device), cfg=cfg, top_k=900, top_p=0.96, g_seed=seed, more_smooth=more_smooth)"?

When running the code, it keeps producing the same images for each class!
I do not know what is causing the problem!

**Comment by Jvanlo** [comment]
Did you modify the seed?

**Comment by moeinheidari7829** [comment]
Yes, I solved it. Thank you very much.
